samples/contrib/versioned-pipeline-ci-samples/kaggle-ci-sample/download_dataset/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.7
```

Comment:
Perhaps we can base on google/cloud-sdk here, which also has Python. Then we'll no longer need to install google-cloud-storage.

Reply:
Thanks. Just curious: Is there any document illustrating the relationships and contents of these inherent images, such as google/cloud-sdk and google-cloud-storage?

Reply:
I test the image just now. It seems that this image does not include built-in python client for google cloud(such as google-cloud-storage), which is needed in the example. Though I believe cloud-sdk image can achieve similar function by using another code script,  I think it would be more convenient to keep the original solution of installing google-cloud-storage in this sample.

Reply:
Got it. Thanks